index.html

```diff
@@ -8,6 +8,10 @@
       content="width=device-width, initial-scale=1.0, minimum-scale=1.0, user-scalable=no"
     />
     <title>Snapshot</title>
+    <meta
+      name="description"
+      content="Snapshot is an off-chain voting platform that allows DAOs, DeFi protocols, or NFT communities to vote easily and without gas fees."
```

Comment:
Let's try to make it more googlable:

```suggestion
      content="Snapshot is an off-chain voting platform that allows DAOs, DeFi protocols, or NFT communities to participate in the decentralized governance easily and without gas fees."
```